### PR TITLE
fix broken photo link

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
fix broken photo link

### CODE REVIEW

**Comment by LiZhenCheng9527** [comment]
cancel /hold

File: content/en/docs/architecture/architecture.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ type: docs
 
 ---
 
-![image](images/kmesh-arch.png)
+![image](/docs/architecture/architecture.svg)
```
Comment (hzxuzhonghu) [inline content/en/docs/architecture/architecture.md:14]: @nlgwcy can you provide arch graph

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**